editor/plugins/editor_plugin.h

```diff
@@ -185,6 +183,7 @@ class EditorPlugin : public Node {
 	virtual String get_plugin_version() const;
 	virtual void set_plugin_version(const String &p_version);
 	virtual bool has_main_screen() const;
+	virtual String get_workspace_display_name() const;
```

Comment:
This can just be called `get_display_name`. 
I don't think we call them 'Workspaces' anywhere in the API, only a couple places in the documentation and inconsistently.

Reply:
My consideration is that a plugin's "display name" can also be understood as the name shown in the Project Settings dialog's Plugins tab. More precise naming creates less misunderstandings and makes the API more intuitive.

We did not explicitly mention the type of these buttons. The type was described by enumeration: the "2D", "3D", "Script", "Game", and "AssetLib" buttons.

https://github.com/godotengine/godot/blob/42c7f144225fa3992ceda284ad8f8edcbd1e7455/doc/classes/EditorPlugin.xml#L263

But the area is called "the workspace selector".

https://github.com/godotengine/godot/blob/42c7f144225fa3992ceda284ad8f8edcbd1e7455/doc/classes/EditorPlugin.xml#L332